packages/office-ui-fabric-react/src/components/Facepile/Facepile.base.tsx

```diff
@@ -190,14 +190,19 @@ export class FacepileBase extends BaseComponent<IFacepileProps, {}> {
     }
 
     const personaNames: string = personasOverflow.map((p: IFacepilePersona) => p.personaName).join(', ');
+    const overflowButtonTitle: string | undefined = overflowButtonProps && overflowButtonProps.title;
     const numPersonasNotPictured: number = Math.max(personasOverflow.length, 0);
 
     const { _classNames } = this;
 
     return (
-      <FacepileButton {...overflowButtonProps} ariaDescription={personaNames} className={_classNames.descriptiveOverflowButton}>
+      <FacepileButton
+        {...overflowButtonProps}
+        ariaDescription={overflowButtonTitle ? undefined : personaNames}
+        className={_classNames.descriptiveOverflowButton}
+      >
         <PersonaCoin
-          title={personaNames}
+          title={overflowButtonTitle ? undefined : personaNames}
```

Comment:
Hmmm, shouldn't this be:

```tsx
title={overflowButtonTitle ? overflowButtonTitle : personaNames}
```

? Perhaps I need ☕️.

Reply:
Oh _I think_ I understand... `title` is provided via `overflowButtonProps` and applied on that element given the object spread here:

https://github.com/OfficeDev/office-ui-fabric-react/pull/7220/files#diff-728df8eba22c2831d1f3a873fd0abc13R200

Reply:
Feel free to close this thread if my understanding is correct. It _may_ be worth adding an in-line comment since it stumped me for a few minutes. Up to you.